File: model/src/main/scala/ru/mipt/acsl/decode/model/domain/impl/registry/LanguageImpl.scala
package ru.mipt.acsl.decode.model.domain.impl.registry

/**
  * @author Artem Shein
  */
private class LanguageImpl(name: ElementName, namespace: Namespace, val isDefault: Boolean, info: Option[String])
  extends AbstractNameNamespaceOptionalInfoAware(name, namespace, info) with Language

File: model/src/main/java/ru/mipt/acsl/decode/model/registry/LanguageImpl.java
package ru.mipt.acsl.decode.model.registry;

/**
 * Created by metadeus on 06.06.16.
 */
public class LanguageImpl implements Language {

    private final String code;

    @Override
    public String code() {
        return code;
    }

    LanguageImpl(String code) {
        this.code = code;
    }
}
